andela/bp-esa-backend
# Refactored non route handler methods to not expect req and res

#### What does this PR do?
Cleans up and refactors methods to require the right parameters

#### Description of Task to be completed?
Refactor methods that are not directly handling http requests to not have the http requests as parameters

#### How should this be manually tested?
1. Checkout `ch-refactor-non-route-handlers-169044711`
2. Run `yarn test`. All tests should pass
3. Check the `AutomationsController`, the `req` and `res` objects are not passed down further to other methods



#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/169044711
